src/update_engine_ie9.jsx

```diff
@@ -64,8 +64,10 @@ export function update() {
  */
 export const requestAsyncUpdate = _.debounce(updateAfterDelay, 0);
 
-// update on window resize
-window.addEventListener('resize', _.debounce(requestAsyncUpdate, 16));
+if (typeof window !== 'undefined') {
```

Comment:
It is also being required in all IE9 components. Probably it makes sense to add this listener in `index`? Or maybe export a function `startListeners` or similar that can be called on `index` if we're in a DOM and we're simulating flexbox.